grader_support/gradelib.py

```diff
@@ -253,7 +250,7 @@ def _count_tokens(code, string):
 
     try:
         for ttyp, ttok, __, __, __ in _tokens(code):
-            if ttyp in (tokenize.COMMENT, tokenize.STRING):
+            if ttyp (COMMENT, STRING):
```

Comment:
@blarghmatey do you know what's edX's style preference might be here, for `tokenize.COMMENT` vs. `COMMENT` ?

Reply:
'm not sure what they're preferred style is, assuming that they even have a consistent style guide...

Reply:
Also, I think this needs the `in` statement as I don't think this will work.